packages/live-preview-sdk/package.json

```diff
@@ -44,7 +44,7 @@
   },
   "homepage": "https://github.com/contentful/live-preview#readme",
   "dependencies": {
-    "@contentful/live-preview-updates": "0.1.0-alpha.21",
+    "@contentful/live-preview-updates": "0.1.0-alpha.22",
```

Comment:
we can't use that one here, as it's an internal package

Reply:
let's do the validation only on the web app and if the validation fails there, we can send a message back to the SDK

Reply:
yeah i can see that it breaks the CI as well,  we'll have to duplicate the validation logic here until we migrate live updates into the web app then

Reply:
No need to duplicate I think,
the logic is already in the web app, if the validation fails, we just send a SUBSCRIPTION_FAILED message back to the LP SDK and log an error with `debug`

Reply:
- Send subscription to the web app
- Web app validates the data/config with @contentful/live-preview-updates
- If successful -> all good live updates should be working
- If failed -> send post message with subscription failed back to the SDK (and maybe the reason)

Reply:
great idea, I've created a follow up [ticket](https://contentful.atlassian.net/browse/TOL-1872?atlOrigin=eyJpIjoiNDY2ZjMyZGY0NzMyNGFiMTg1ODY1MjgwOGM3MzQ0OTkiLCJwIjoiaiJ9) to remove the duplication so that this PR's scope doesn't keep increasing